.release.cloudbuild.yaml

```diff
@@ -141,10 +141,10 @@ steps:
   waitFor: ['PullMetadataEnvoy']
 
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['pull', 'gcr.io/ml-pipeline/minio:RELEASE.2019-08-14T20-37-41Z']
+  args: ['pull', 'gcr.io/ml-pipeline/minio:RELEASE.2019-08-14T20-37-41Z-license-compliance']
   id:   'PullMinio'
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['tag', 'gcr.io/ml-pipeline/minio:RELEASE.2019-08-14T20-37-41Z', 'gcr.io/ml-pipeline/google/pipelines/minio:$TAG_NAME']
+  args: ['tag', 'gcr.io/ml-pipeline/minio:RELEASE.2019-08-14T20-37-41Z-license-compliance', 'gcr.io/ml-pipeline/google/pipelines/minio:$TAG_NAME']
```

Comment:
gcloud can add tags to an image without pulling it locally. I wonder whether it can work with external images...

Reply:
What use case do you have in mind? I recall the doc says it only works with gcr

Reply:
Did a quick search and found this: https://stackoverflow.com/questions/37134929/how-to-tag-image-in-docker-registry-v2#

I don't think there's an easy builtin way to do that with `docker tag`.
But using gcloud sdk image to directly tag remote image would be an improvement. https://github.com/GoogleCloudPlatform/cloud-sdk-docker

Anyway, I think this is out of scope for this PR. We can do that later.